core/src/test/scala/unit/kafka/zk/AdminZkClientTest.scala

```diff
@@ -194,6 +199,8 @@ class AdminZkClientTest extends QuorumTestHarness with Logging with RackAwareTes
    */
   @Test
   def testTopicConfigChange(): Unit = {
+    TestUtils.deleteControllerFromZk(zkClient)
```

Comment:
I can understand we need to createController in ZK first to set configs. But I don't understand why we should delete the controller from ZK here? Might need some comments for it.